src/Styles/BackOfficerUsersTable.css

```diff
@@ -0,0 +1,6 @@
+.BackofficeUsers{
+    background-color:whitesmoke;
+}
+.BackofficeUsersThead{
```

Comment:
this styles should go on table style and the names should be more generic. Try to reuse any style if exists